1-exercises/C-array-get-set/exercises2.js

```diff
@@ -8,6 +8,8 @@
 
 let numbers = [1, 2, 3]; // Don't change this array literal declaration
 
+numbers[3] = numbers.push(4)
+//The first value is 1???
```

Comment:
Good spot I'll forward that observation. But if it wasn't  a `1` how would you change its value?